src/profiler.ts

```diff
@@ -106,45 +106,20 @@ function getServerResponseBackoff(body: object): number | undefined {
   const b = body as any;
   if (b.error && b.error.details && Array.isArray(b.error.details)) {
     for (const item of b.error.details) {
-      if (
-        typeof item === 'object' &&
-        item.retryDelay &&
-        typeof item.retryDelay === 'string'
-      ) {
-        const backoffMillis = parseDuration(item.retryDelay)!;
-        if (backoffMillis > 0) {
-          return backoffMillis;
+      if (typeof item === 'object' && item.retryDelay) {
+        const retryDelay: ms.StringValue = item.retryDelay;
+        if (typeof retryDelay === 'string') {
+          const backoffMillis = ms(retryDelay)!;
+          if (backoffMillis > 0) {
+            return backoffMillis;
+          }
         }
       }
     }
   }
   return undefined;
 }
 
-/**
```

Comment:
This is dead code which was missed in https://github.com/googleapis/cloud-profiler-nodejs/pull/402